vale/fixtures/Grafana/Wordlist/testvalid.md

```diff
@@ -1,7 +1,15 @@
+# Promtail
+
 Instead use:
 
 - Promtail
 
 Also OK:
 
 - [lambda-promtail](https://github.com/grafana/loki/tree/main/tools/lambda-promtail)
+
+# Data Firehose
+
+Instead use:
+
+- Data Firehose
```

Comment:
Should this be `Amazon Data Firehose`?  https://aws.amazon.com/firehose/

Reply:
Great question. Our guidance is `Amazon <PRODUCT>` in the first instance and then just `<PRODUCT>` thereafter: https://grafana.com/docs/writers-toolkit/write/style-guide/capitalization-punctuation/#amazon-products and for that we have: https://github.com/grafana/writers-toolkit/blob/main/vale/Grafana/AmazonProductNames.yml